src/ledger/LedgerManagerImpl.cpp

```diff
@@ -1373,6 +1388,7 @@ LedgerManagerImpl::setLastClosedLedger(
     LedgerTxn ltx(mApp.getLedgerTxnRoot());
     auto header = ltx.loadHeader();
     header.current() = lastClosed.header;
+    auto lv = header.current().ledgerVersion;
```

Comment:
nit: you're accessing an uncommitted half-valid ltx here (but at the right time). It'd be slightly safer to just extract the version from `lastClosed`